app/assets/javascripts/backbone/views/calendar_view.coffee

```diff
@@ -8,8 +8,8 @@ class MS.Views.CalendarView extends Backbone.View
     @$calendar = @$('#calendar')
     @stepModal = params.stepModal
     defaultCalendarSettings =
-      # Changes the default event render to load in html rather than title only
-      eventRender: @eventRender.bind(context)
+      # eventRender: @eventRender.bind(context)
```

Comment:
eventRender can be removed

This feature utilizes the underlying fullCalendar attributes and classes. Therefore eventAfterRender is used so that those classes can be populated before the content for the event is manipulated.